modules/release-notes/pages/index.adoc

```diff
@@ -21,19 +24,21 @@ The following is a list of new features and improvements:
 === TigerGraph Insights
 
 * Added support for multi-user editing of the same application.
-* Added the ability for users to save widgets independently instead of tying them to an application.
+* Added the ability for users to xref:insights:widgets:index.adoc#_save_a_widget[save a widget] independently instead of tying them to an application.
```

Comment:
```suggestion
* Added the ability for users to xref:insights:widgets:index.adoc#_save_a_widget[save a widget] independently instead of tying it to an application.
```